Library/Formula/go.rb

```diff
@@ -22,7 +22,8 @@ class Go < Formula
 
   resource "gotools" do
     url "https://go.googlesource.com/tools.git",
-    :revision => "d02228d1857b9f49cd0252788516ff5584266eb6"
+    :branch => "release-branch.go#{version}",
```

Comment:
Unfortunately that'll use the `version` from the resource. Instead make a `go_version = version` outside of the `resource do` and then use `#{go_version}` in here.